frontend/src/config/ServerConfig.ts

```diff
@@ -51,6 +51,7 @@ const computeValidDurations = (cfg: ComputedServerConfig) => {
 // than may not be providedby/set on the server.
 const defaultServerConfig: ComputedServerConfig = {
   accessibleNamespaces: [],
+  ambientProfile: false,
```

Comment:
I think this is another one where "profile" isn't an accurate term to be used. Ambient could be installed in other istio profiles... I would prefer something like "ambientEnabled" or "ambientSupported".

Reply:
Changed for AmbientEnabled